haskell-src/Concordium/Types/Parameters.hs

```diff
@@ -341,6 +342,17 @@ module Concordium.Types.Parameters (
     DelegationChainParameters (..),
     delegationChainParameters,
 
+    -- * Finalization committee parameters
+    FinalizationCommitteeParameters (..),
+    -- The minimum number of bakers in the finalization committee.
```

Comment:
I really don't think this phrasing is good since it is not a minimum as conventionally understood. There can be fewer if there aren't enough bakers.

Please spare an extra sentence or two to explain.